UniTrain/utils/classification.py

```diff
@@ -1,14 +1,37 @@
 import os
-import UniTrain
 from torch.utils.data import DataLoader
 from torchvision import transforms
-from UniTrain.dataset.classification import ClassificationDataset
+from ..dataset.classification import ClassificationDataset
```

Comment:
To follow the repository format and make the training generalized 
import UniTrain
from UnTrain.dataset.classification import ClassificationDataset